osu.Game/Database/RealmLive.cs

```diff
@@ -125,4 +165,12 @@ private T retrieveFromID(Realm realm, Guid id)
 
         public override string ToString() => PerformRead(i => i.ToString());
     }
+
+    public class RealmLive
```

Comment:
Maybe a static class? Named `RealmLiveStatistics`? I don't like that this is being inherited from and constructable.